src/api/migrations/index.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import { join as joinPath } from 'path';
+
+import { sync as syncGlob } from 'glob';
+import { moveSync } from 'fs-extra';
+
+import type { MigratorType } from '../../types';
+
+const migrations = ([
+  (config) =>
```

Comment:
This is a specific migration that seems a little "hidden" in here :-) how about extracting it to an own file?